### PR TITLE
[ExoBundle] Switch to webpack/bower

### PR DESCRIPTION
**Pull request by stefk**
- exercise player doesn't need assetic anymore
- most third-party libs are managed by bower
- minor fixes/enhancements

### CODE REVIEW

File: plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function UnsafeFilter($sce) {
```
Comment (ngodfraind) [inline plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js]: This filter already exists in the core-bundle (module/html-truster).

File: plugin/exo/Resources/modules/common/Filters/DateStringFilter.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * @returns {function}
  * @constructor
  */
-var DateStringFilter = function DateStringFilter($filter) {
+function DateStringFilter($filter) {
```
Comment (ngodfraind) [inline plugin/exo/Resources/modules/common/Filters/DateStringFilter.js]: Maybe move this into the corebundle as it's pretty generic and could be usefull for everyone

File: plugin/exo/Resources/modules/common/Filters/RouterFilter.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * @returns {function}
  * @constructor
  */
-var RouterFilter = function RouterFilter() {
+function RouterFilter() {
```
Comment (ngodfraind) [inline plugin/exo/Resources/modules/common/Filters/RouterFilter.js]: This filter already exists in the core bundle (modules/fos-js-router)

File: plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js (hunk 1 above)
Comment (Elorfin) [inline plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js]: Yes.
The remove of the `Common` module in favor of Core will come in a next pull-request.
The goal here is just to initialize webpack, not refactorize the whole code base.

File: plugin/exo/Resources/modules/common/module.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      DateStringFilter
+  ])
+  .filter('path', [
+      RouterFilter
+  ])
+  .filter('simple_type', [
+      'CommonService',
+      SimpleTypeFilter
+  ])
+  .filter('unsafe', [
```
Comment (ngodfraind) [inline plugin/exo/Resources/modules/common/module.js]: Some of these filters can be fetched from the core-bundle.

File: plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js (hunk 1 above)
Comment (ngodfraind) [inline plugin/exo/Resources/modules/common/Filters/UnsafeFilter.js]: Ok, you can ignore these comments then.

**Comment by purplefish32** [comment]
LGTM

**Comment by stefk** [comment]
LGTM

**Pull request merged**